schema.ts

```diff
@@ -1232,6 +1232,43 @@ export interface ObjectSchemaDefinition<K extends string, L extends string>
   // TODO(dweitzman): Only support options in the typing when the codaType is ValueHintType.SelectList.
 }
 
+/**
+ * Represents a permission in the external system
```

Comment:
Is this only external systems?  Do we need to define what we mean by external system here?  Not sure what we want to say publicly.

Reply:
That is a good question I guess it just means a permission on the entity instead of external system which is technically optional in a pack